adapters/sharethrough/butler.go

```diff
@@ -91,14 +92,20 @@ func (s StrOpenRTBTranslator) requestFromOpenRTB(imp openrtb.Imp, request *openr
 			Width:              width,
 			InstantPlayCapable: s.Util.canAutoPlayVideo(request.Device.UA, s.UserAgentParsers),
 			TheTradeDeskUserId: userInfo.TtdUid,
+			SharethroughUserId: request.User.BuyerUID,
```

Comment:
Mathieu, your adapter went live but kept panic'ing here because of a NULL pointer de reference. Please implement a workaround. We also think a test case in `TestSuccessResponseToOpenRTB(t *testing.T)` (or anywhere in `butler_test.go` for that matter) to test for a `nil` value would be great.